README.md

```diff
@@ -128,6 +130,7 @@ https://oneapi.io/ for more information.
 ### Autonomous Systems
 
 * [Alice](https://github.com/intel/dffml/tree/alice/entities/alice/) - We are writing a tutorial for an open source project on how we build an AI to work on the open source project as if she were a remote developer. Bit of a self fulfilling prophecy but who doesn't love an infinite loop now and again.
+* [FreeChat](https://github.com/lm-sys/FastChat) - FastChat is an open platform for training, serving, and evaluating large language model based chatbots.
```

Comment:
I think you meant FastChat here

Reply:
Yes, that's right - thanks for that - I'll fix it. I must have had Free Software on my mind. :)